cypress/e2e/CreateNewAnalysis.cy.ts

```diff
@@ -0,0 +1,86 @@
+
+describe('Testing SignUp Page', () => {
+    it('Visits the Feeds Page', () => {
+      cy.visit('http://localhost:3000/feeds')
+  
+    })
+
+    it('Clicks on Create New Analysis', () => {
+        cy.get('.create-feed-button')
+        .click()
+    })
+
+    it('Types in the Feed Name', () => {
+        cy.get('input#feed-name').type('MRI Study')
```

Comment:
This comment does not need to be addressed for this PR but please think about them 

Consider having the string "MRI Study" include a random component. Then write an _assertion_ to check whether or not the feed title was actually set correctly by the ChRIS_ui.

For example, I am writing some tests for a project I am working on, [chrs](https://github.com/FNNDSC/chrs). Its automatic tests create pipelines with random names and random descriptions. These are just two random words.

![screenshot](https://user-images.githubusercontent.com/20404439/177064954-46ff7113-4860-4cf5-8371-9946ca16d357.png)